examples/Aggregator/Server/Server.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Protobuf Include="..\Proto\aggregate.proto" GrpcServices="Server" Link="Protos\aggregate.proto" />
```

Comment:
We should remove GrpcServices attributes. The default here is Server.

Reply:
How is it defaulted? The template adds `GrpcServices="Server"`

Reply:
Oops, brainfart, GrpcServices="Server" is needed.